### PR TITLE
Add documentation for pi to run multi interface cyclone file and docu…

### PR DESCRIPTION
**Pull request by justinIRBT**
- Add documentation for pi to run multi interface cyclone file
- Document don't route laptop config to work with multiple interface
Hopefully default instruction flow for pi setup now takes people through the multi config setup that should yield them a ROS 2 topic list success for the multi-interface case.

### CODE REVIEW

File: docs/setup/pi4.md [hunk 1]
```diff
@@ -64,4 +64,10 @@ The first boot may take a few minutes. (It may help to have a monitor and keyboa
 1. Log out and log back in. Once you do, test things out with a `ros2 topic list`.
 A full Create® 3 API description can be found [here](../../api/ros2).
 
+    !!! attention
+        **If you are using CycloneDDS (Galactic default), your Pi is running with multiple network interfaces (usb0 to talk to robot and wlan0 to talk to laptop).**
```
Comment (shamlian) [inline docs/setup/pi4.md:68]: your Pi _may be_ running with

File: docs/setup/jetson.md [hunk 2]
```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 3. Set the default network interface by setting Cyclone DDS configuration.
 
         export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+    !!! attention
+        **If you are using CycloneDDS (Galactic default) and want the Jetson to talk to the robot over USB and a laptop via wifi, you have a multiple interface situation.**
```
Comment (shamlian) [inline docs/setup/jetson.md:117]: you have a multiple interface situation -> you will need to take extra steps to use multiple interfaces.

File: docs/setup/jetson.md [hunk 3]
```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
...
+        **You will need to export a path on the Jetson to an xml config file that registers those interfaces in the CYCLONEDDS_URI.**
+        **See [CycloneDDS Multiple Network Interfaces](../xml-config/#cyclonedds).**
+        **Note the differences in Jetson USB and Wifi interface names from the documentation.**
```
Comment (shamlian) [inline docs/setup/jetson.md:120]: Wi-Fi

File: docs/setup/pi4.md (hunk 1 above)
Comment (shamlian) [reply docs/setup/pi4.md:68]: Pi -> Raspberry Pi®

File: docs/setup/pi4.md [hunk 4]
```diff
@@ -64,4 +64,10 @@ The first boot may take a few minutes. (It may help to have a monitor and keyboa
...
+        **You will need to export a path on the Pi to an xml config file that registers those interfaces in the CYCLONEDDS_URI.**
```
Comment (shamlian) [inline docs/setup/pi4.md:69]: Pi ->  Raspberry Pi®

**Review by shamlian — CHANGES_REQUESTED** [review]
Some minor changes. I would also suggest switching the footnote style on the Pi 4 page to use the footnote plugin (see electrical.md for an example) but this is not strictly necessary. Otherwise, looks good.

**Review by shamlian — APPROVED** [review]

File: docs/setup/jetson.md [hunk 5]
```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 3. Set the default network interface by setting Cyclone DDS configuration.
 
         export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+    !!! attention
+        **If you are using CycloneDDS (Galactic default) and want the Jetson to talk to the robot over USB and a laptop via Wi-Fi, you will need to take extra steps to use multiple interfaces.**
```
Comment (alsora) [inline docs/setup/jetson.md:117]: "to use multiple interfaces" --> "to setup CycloneDDS to use multiple network interfaces"

File: docs/setup/jetson.md [hunk 6]
```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
...
+        **You will need to export a path on the Jetson to an xml config file that registers those interfaces in the CYCLONEDDS_URI.**
```
Comment (alsora) [inline docs/setup/jetson.md:118]: I find this sentence a little bit confusing.
Maybe rephrase as:
"You will need to create on the Jetson a CycloneDDS XML configuration file and set the CYCLONEDDS_URI environment variable to its path"

File: docs/setup/xml-config.md [hunk 7]
```diff
@@ -90,6 +90,18 @@ For example `usb0` and `wlan0` in this example.
 
 Note that the specified network interfaces must be already active when the ROS 2 process is started.
 
+!!! attention
+    **If the robot is running with a Compute Board like a [Raspberry Pi®](../pi4) or an [NVIDIA® Jetson™](../jetson) connected, then the robot is using a multiple interface CycloneDDS config file to communicate both over usb0 and wlan0.**
```
Comment (alsora) [inline docs/setup/xml-config.md:94]: "connected" --> "connected via usb"

**Review by alsora — APPROVED** [review]
